chore: make text consistent with new SDK version (#169)
KILTprotocol/docs: docs/develop/03_workshop/06_attestation.md

In this section, your <span className="label-role attester">Attester</span> will receive and process a `Credential` where you'll

- Attest or deny it;
- Store the attestation information on the chain;

## Attest a Credential

The `attestCredential` function loads the account and DID of the <span className="label-role attester">Attester</span>.
When everything is prepared, we can issue an attestation for the credential we received from the <span className="label-role claimer">Claimer</span>.

> The credential becomes valid once it is attested onto the chain.

Reviewer: ```suggestion
The credential becomes valid once an attestation exists for it on the blockchain.
```

Author: Do you think this language is better? An attestation could also be revoked, making the credential invalid.

Reviewer 2: I think Antonio's version is better sounding.

> The credential is considered to be valid from the time it is attested on chain until the time it is revoked.

Reviewer 2: This reads better
